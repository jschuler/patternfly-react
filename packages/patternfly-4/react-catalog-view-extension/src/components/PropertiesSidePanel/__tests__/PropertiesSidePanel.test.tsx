// Auto-generated do not edit

import * as React from 'react';
import { shallow } from 'enzyme';
import { PropertiesSidePanel } from '../PropertiesSidePanel';

describe('PropertiesSidePanel test', () => {
  it('PropertiesSidePanel should match snapshot', () => {
    const view = shallow(
      <PropertiesSidePanel
        className={"''"}
				children={<>ReactNode</>}
      />);
    expect(view).toMatchSnapshot();
  });
});
