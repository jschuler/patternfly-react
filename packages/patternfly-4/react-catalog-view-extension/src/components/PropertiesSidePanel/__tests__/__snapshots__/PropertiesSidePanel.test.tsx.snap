// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PropertiesSidePanel test PropertiesSidePanel should match snapshot 1`] = `
<div
  className="properties-side-panel-pf ''"
>
  ReactNode
</div>
`;
